lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1106,6 +1106,16 @@ sub save_user {
            my $template = LedgerSMB::Template->new_UI(
                $request,
                template => 'setup/new_user',
+               hiddens  => {
```

Comment:
Looking at https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Template.pm#L13 and https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Template.pm#L115, `new_UI` doesn't take a `hiddens` argument.
This needs changing before it can be merged.

Reply:
Unused hiddens removed.